App/Components/ContactListItem/index.js

```diff
@@ -1,12 +1,13 @@
-import React from 'react';
-import {View, Text, Platform, TouchableOpacity, Linking} from 'react-native';
+import React, { useState } from 'react';
+import {View, Text, Platform, TouchableOpacity, Linking, Modal} from 'react-native';
 import FastImage from 'react-native-fast-image';
 import styles from './styles';
 import {Fonts, Colors, ApplicationStyles} from '../../Themes';
 import Icon from 'react-native-vector-icons/FontAwesome5';
+import IconFont from 'react-native-vector-icons/FontAwesome';
 
 export default data => {
-  const {item, onPressedCell, pictures} = data;
+  const {item, onPressedCell, pictures, favoritos, setFavoritos, deleteFavoritos} = data;
```

Comment:
I recommend activating the `cSpell` and `eslint` tools help to write some words correctly.